src/main/java/edu/hm/hafner/coverage/CoverageParser.java

```diff
@@ -166,7 +166,7 @@ public final TreeStringBuilder getTreeStringBuilder() {
      *         the logger to write messages to
      *
      * @return the root of the created tree
-     * @throws ParsingException
+     * @throws RuntimeException
```

Comment:
It's due to `GoParser` extending `CoverageParser` and overriding the `parseReport()` method. `ParsingException` comes from `SecureXmlParserFactory`, but because Go coverage isn't in XML format, it didn't felt misleading to have it throw an XML Parsing exception or a child exception of it. So I created a new exception that was a child at the next level, which was the`RuntimeException`